src/services/content/hooks/search.js

```diff
@@ -0,0 +1,24 @@
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+
+exports.before = {
+	all: [
+		authenticate('jwt'),
+	],
+	find: [],
```

Comment:
Please add a comment if it is ok, that no scope hooks exist.

Reply:
same here, @kremer-io